cypress/e2e/tests/Providers/CreateProvider.cy.ts

```diff
@@ -9,7 +9,7 @@ describe('Providers list view', () => {
   });
 
   it('has a add-provider button', () => {
-    // find and click the create provider button
-    cy.findByTestId('add-provider-button').should('exist').click();
+    // find and click the create provider button (the 1st one since there are two occurrences)
+    cy.findAllByTestId('add-provider-button').should('exist').click();
```

Comment:
@sgratch why do we need `find all` approach here? Is the fix from https://github.com/kubev2v/forklift-console-plugin/pull/685 no longer working?

Reply:
@rszwajko @sgratch i must have re intruduced the multiple id somewhere ...

this PR should fix it:
https://github.com/kubev2v/forklift-console-plugin/pull/701